v/vivy.md

```diff
@@ -0,0 +1,8 @@
+# Vivy
+
+![alt text](vivy.jpg) 
```

Comment:
Please replace this alt text with the artist's name, or describe what is happening in the photo.